### PR TITLE
Add Mobility category and the following companies

### PR DESCRIPTION
**Pull request by jcoelho93**
* Amenitiz
* BVNK
* Epos Now
* FAIRTIQ
* Glia
* PagerDuty
* Stream
* SwissBorg

### CODE REVIEW

**Review by marmelo — COMMENTED** [review]
Still need to validate the new companies, but come preliminar comments.

File: README.md [hunk 1]
```diff
@@ -307,7 +315,7 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [Criteria Marathon](https://www.criteriamarathon.com/) | Software Engineering and Information Technologies in the Medical Sector. | `Porto` |
 | [iLoF](https://ilof.tech/) | Revolutionizing Alzheimer’s drug discovery using bio-photonics and AI. | `Porto` |
 | [LetsGetChecked](https://www.letsgetchecked.com/) [:rocket:](https://boards.eu.greenhouse.io/letsgetchecked/) | The future of healthcare starts at home. | `Porto` `Remote` |
-| [Nutrium](https://www.nutrium.com/) [:rocket:](https://nutrium.recruitee.com/) | Nutrition platform for improving eating habits around the world. | `Braga` `Lisboa` `Remote` |
+| [Nutrium](https://www.nutrium.com/) [:rocket:](https://nutrium.factorialhr.pt/) | Nutrition platform for improving eating habits around the world. | `Braga` `Lisboa` `Remote` |
```
Comment (afonsojramos) [inline README.md:318]:
```suggestion
| [Nutrium](https://www.nutrium.com/) [:rocket:](https://nutrium.recruitee.com/) | Nutrition platform for improving eating habits around the world. | `Braga` `Lisboa` `Remote` |
```

**Review by afonsojramos — COMMENTED** [review]
Looks good, just apply the necessary changes 👌

File: README.md [hunk 2]
```diff
@@ -321,6 +329,13 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [Malvern Panalytical](https://www.malvernpanalytical.com/) [:rocket:](https://spectris.wd3.myworkdayjobs.com/Malvern_Panalytical_Careers) | Laboratory analytical instruments design and manufacturing. | `Maia` |
 | [Relex Solutions](https://www.relexsolutions.com/) [:rocket:](https://www.relexsolutions.com/careers/) | The market-leading supply chain & retail planning platform. | `Lisboa` |
 
+## Mobility :bike:
+
+| Company | Description | Locations |
+| :------ | :---------- | :-------- |
+| [FAIRTIQ](https://fairtiq.com/) [:rocket:](https://fairtiq.com/en/careers) | Public transport electronic tickets | `Remote` |
+| [Stratio](https://stratioautomotive.com/) [:rocket:](https://careers.stratioautomotive.com/) | Predictive fleet maintenance platform | `Lisboa` `Coimbra` `Remote` |
```
[the remark this answers is not in the document]
Comment (jcoelho93) [reply README.md:336]: I hadn't noticed. Moved to Mobility category

**Comment by marmelo** [comment]
> @marmelo @vpereira01 what do you think here?

I have mixed feelings with companies that are fully Remote and not based in PT.
If we allow them to enter the list, virtually all companies in the world are eligible, thus defying the purpose of this list.

What do you think?

**Comment by vpereira01** [comment]
> > @marmelo @vpereira01 what do you think here?
> 
> I have mixed feelings with companies that are fully Remote and not based in PT. If we allow them to enter the list, virtually all companies in the world are eligible, thus defying the purpose of this list.
> 
> What do you think?

From the previous discussion, the rule has been full remote is OK if the company has a policy to recruit from Portugal. Preferably the company has already employees with residence in Portugal.

Most remote companies have restrictions from where they recruit so this approach reduces the risk of too many companies.

**Comment by marmelo** [comment]
> > > @marmelo @vpereira01 what do you think here?
> > 
> > 
> > I have mixed feelings with companies that are fully Remote and not based in PT. If we allow them to enter the list, virtually all companies in the world are eligible, thus defying the purpose of this list.
> > What do you think?
> 
> From the previous discussion, the rule has been full remote is OK if the company has a policy to recruit from Portugal. Preferably the company has already employees with residence in Portugal.
> 
> Most remote companies have restrictions from where they recruit so this approach reduces the risk of too many companies.

I am OK with this rule.


> > > @marmelo @vpereira01 what do you think here?
> > 
> > 
> > I have mixed feelings with companies that are fully Remote and not based in PT. If we allow them to enter the list, virtually all companies in the world are eligible, thus defying the purpose of this list.
> > What do you think?
> 
> From the previous discussion, the rule has been full remote is OK if the company has a policy to recruit from Portugal. Preferably the company has already employees with residence in Portugal.
> 
> Most remote companies have restrictions from where they recruit so this approach reduces the risk of too many companies.

I agree with these rules.

TL;DR it is OK to have companies fully remote that comply with any of:
1. Careers page explicitly seeks remote people from PT / EU;
2. Some PT engineers are already working remotely in the company.

If we all agree with this, I have added my PR comments.
All companies are OK except Amenitiz.

@jcoelho93 can you tell us why should Amenitiz be part of this list?

Thanks!

**Comment by afonsojramos** [comment]
> TL;DR it is OK to have companies fully remote that comply with any of:
> 
> Careers page explicitly seeks remote people from PT / EU;
> Some PT engineers are already working remotely in the company.

I'll add this to our Contributing page!

File: README.md [hunk 3]
```diff
@@ -223,13 +225,16 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
+| [Amenitiz](https://www.amenitiz.com/) [:rocket:](https://careers.amenitiz.io/) | Hotel Management Software | `Remote` |
```
[the remark this answers is not in the document]
Comment (jcoelho93) [reply README.md:228]: I've been in a recruiting process with all the companies I've added, that's how I know they are hiring in Portugal. Some may not have employees yet in Portugal but all are open to it.

[the remark this answers is not in the document]
Comment (marmelo) [reply README.md:228]: Thanks @jcoelho93!

**Review by marmelo — APPROVED** [review]

**Comment by marmelo** [comment]
> > TL;DR it is OK to have companies fully remote that comply with any of:
> > Careers page explicitly seeks remote people from PT / EU;
> > Some PT engineers are already working remotely in the company.
> 
> I'll add this to our Contributing page!

Thank you! :pray:

**Comment by vpereira01** [comment]
:+1: tks @jcoelho93
